product_docs/docs/epas/14/epas_inst_linux/installing_epas_using_edb_repository/x86_amd64/epas_ubuntu18_x86.mdx

```diff
@@ -3,48 +3,42 @@ navTitle: Ubuntu 18.04
 title: Installing EDB Postgres Advanced Server on Ubuntu 18.04 x86_64
 ---
 
-Before you begin the installation process, log in as superuser.
+## Prerequisites
 
-```shell
-# To log in as a superuser:
-sudo su -
-```
+Before you begin the installation process:
 
-## Set up the repository
+- Set up the repository
 
-Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step. If you do need to set up the repository, you must register with EDB. To receive credentials for the EDB repository, visit: [Repository Access Request](https://www.enterprisedb.com/repository-access-request).
+  Setting up the repository is a one-time task. If you have already set up your repository, you do not need to perform this step.
 
-```shell
-#  Set up the EDB repository
-echo "deb [arch=amd64] https://apt.enterprisedb.com/$(lsb_release -cs)-edb/ $(lsb_release -cs) main" > /etc/apt/sources.list.d/edb-$(lsb_release -cs).list
+  To set up the repository, go to [EDB repositories](https://www.enterprisedb.com/repos-downloads) and follow the instructions provided there.
 
-#  Replace '<USERNAME>' and '<PASSWORD>' below with
-#  your username and password for the EDB repositories:
-echo "machine apt.enterprisedb.com login <USERNAME> password <PASSWORD>" > /etc/apt/auth.conf.d/edb.conf
+- Other prerequisites
 
-#  Add support for secure APT repositories:
-apt-get -y install apt-transport-https
+  ```shell
+  #  Add support for secure APT repositories:
+  sudo apt-get -y install apt-transport-https
 
-#  Add the EDB signing key. Substitute your EnterpriseDB credentials
-#  for the '<USERNAME>' and '<PASSWORD>' placeholders.
-wget -q -O - https://<USERNAME>:<PASSWORD>@apt.enterprisedb.com/edb-deb.gpg.key  | sudo apt-key add -
+  #  Add the EDB signing key. Substitute your EnterpriseDB credentials
+  #  for the '<USERNAME>' and '<PASSWORD>' placeholders.
+  wget -q -O - https://<USERNAME>:<PASSWORD>@apt.enterprisedb.com/edb-deb.gpg.key  | sudo apt-key add -
 
-#  Update the repository metadata:
-apt-get update
-```
+  #  Update the repository metadata:
+  sudo apt-get update
+  ```
 
 ## Install the package
 
 ```shell
-apt-get install edb-as<xx>-server
+sudo apt-get install edb-as<xx>-server
```

Comment:
sudo apt-get -y install edb-as<xx>-server

Reply:
@sandeep-edb does the -y option apply to all OSs including SLES 12 and 15?

Reply:
The -y option applies to RedHat and Debian based OSs. On SLES, the option for non-interactive installation is -n